### PR TITLE
Cook Off - Change Cook Off events to global and add to events docs

### PR DESCRIPTION
**Pull request by Dahlgren**
**When merged this pull request will:**
- Broadcast `ace_cookoff_cookOff`, `ace_cookoff_cookOffBox` and `ace_cookoff_engineFire` globally, event wasn't sent locally before
- Add both events to events documentation

### CODE REVIEW

**Comment by Dahlgren** [comment]
I'll do some SP and MP testing tonight

**Comment by PabstMirror** [comment]
`fnc_handleDamage` (where vehicle is local) calls `fnc_cookOff`
`fnc_cookOff` (where vehicle is local) calls the event on all other remote machines
event triggers `fnc_cookOff` to run on all other machines

So this isn't needed for functionality, but would allow the event to be used publicly
(shouldn't cause problems because it's gated with the `isCookingOff` setVar check)

**Comment by commy2** [comment]
pls test, before this deadlocks the game

**Comment by Dahlgren** [comment]
Did some SP testing yesterday which worked fine after fixing the other bug (#7419). I'll try to do the MP part today.

**Comment by PabstMirror** [comment]
```
if (_vehicle getVariable [QGVAR(isEngineSmoking), false]) exitWith {};
_vehicle setVariable [QGVAR(isEngineSmoking), true];
```
this should prevent infinite recursion, but actual testing is always good

**Comment by Dahlgren** [comment]
Seems to work good in MP as well

File: docs/wiki/framework/events-framework.md [hunk 1]
```diff
@@ -97,6 +97,13 @@ MenuType: 0 = Interaction, 1 = Self Interaction
 |`ace_refuel_tick` | [_source, _target, _amount] | Local | Listen | Amount of fuel transferred in a tick
 |`ace_refuel_stopped` | [_source, _target] | Local | Listen | Refueling has stopped
 
+### 2.10 Cook Off (`ace_cookoff`)
+
+| Event Key | Parameters | Locality | Type | Description |
+|----------|---------|---------|---------|---------|---------|
+|`ace_cookoff_cookOff` | [_vehicle] | Global | Listen | Vehicle cook off has started
```
Comment (PabstMirror) [inline docs/wiki/framework/events-framework.md]: no array, just raw object
```suggestion
|`ace_cookoff_cookOff` | _vehicle | Global | Listen | Vehicle cook off has started
```

File: docs/wiki/framework/events-framework.md [hunk 2]
```diff
@@ -97,6 +97,13 @@ MenuType: 0 = Interaction, 1 = Self Interaction
...
+|`ace_cookoff_cookOffBox` | [_box] | Global | Listen | Ammo box cook off has started
```
Comment (PabstMirror) [inline docs/wiki/framework/events-framework.md]:
```suggestion
|`ace_cookoff_cookOffBox` | _box | Global | Listen | Ammo box cook off has started
```

File: docs/wiki/framework/events-framework.md [hunk 3]
```diff
@@ -97,6 +97,13 @@ MenuType: 0 = Interaction, 1 = Self Interaction
...
+|`ace_cookoff_engineFire` | [_vehicle] | Global | Listen | Engine fire has started
```
Comment (PabstMirror) [inline docs/wiki/framework/events-framework.md]:
```suggestion
|`ace_cookoff_engineFire` | _vehicle | Global | Listen | Engine fire has started
```

**Pull request merged**